### PR TITLE
Updating intro modal for small screens

### LINKED ISSUE #456: Hidden Instructions page
**What did you do?**

Signed into eApp

**What did you expect to see?**

Show me this page so I can click to agree with the instructions
screen shot 2018-06-20 at 10 11 52 am

**What did you see instead?**

It took me to the first screen as if I already clicked the "Agree" button on the instructions page

screen shot 2018-06-20 at 10 12 04 am

**Additional Information**

This happens when the width of screen is narrowed. What it looks like to me is that the instructions screen is hidden on narrow windows.

 - What browser and version are you using?
chrome
 - What operating system?
mac

### PR DESCRIPTION
**Pull request by line47**
Fixes #456 

The intro modal now shows for all screen sizes. 👍 

screen shot 2018-07-09 at 4 58 30 pm

screen shot 2018-07-09 at 4 58 02 pm

### CODE REVIEW

File: src/sass/phone.scss [hunk 1]
```diff
@@ -38,7 +38,6 @@
         margin-bottom: 0 !important;
 
         .consent-legal {
-          padding: 0;
           height: 70vh;
```
Comment (line47) [inline src/sass/phone.scss]: Removed this to allow some padding between the modal text and edge of the modal at small screens.

File: src/views/Login/Login.scss [hunk 2]
```diff
@@ -79,7 +79,7 @@
   .auth {
     display: inline-block;
     vertical-align: top;
-    width: 48%;
```
Comment (line47) [inline src/views/Login/Login.scss]: set a max-width instead of width because at small screens things got a little too squished

**Comment by afeld** [comment]
Login screen still needs work, but an improvement!

screen shot 2018-07-09 at 10 37 21 pm

**Comment by line47** [comment]
@afeld yeah I had some more changed lined up but didn't want to add to this PR. :)

**Pull request merged**